README.md

```diff
@@ -1,190 +1,148 @@
 
-And you can get/set those headers by simply changing the property value without worrying of underlying representation
+WinDivert supports access and modification of a variety of TCP/UDP/ICMP attributes out of the box.
 
 ```python
-with Handle(filter="tcp.DstPort == 23 or tcp.SrcPort == 13131") as handle:
-    while True:
-        packet = handle.receive()
-        if packet.meta.is_outbound():
-            packet.dst_port = 13131
-        else:
-            packet.src_port = 23
-        handle.send(packet)
-```
-
-Checkout the test suite for examples of usage.
-
-Any feedback is more than welcome!
-
-TODOs
------
-
-1. ~~Packet modification and reinjection~~
-2. ~~Support for other platforms, at least OSX and linux~~ Discarded. These are out of scope for this binding.
-3. ~~May be a good idea to delegate all the WinDivert methods to Handle instances~~
-4. Clean test code. Tests should be more readable to use as usage example
-5. Implement binding for Asynchronous I/O using WinDivertSendEx and WinDivertRecvEx
-6. ~~Drop support for WinDivert 1.0.x~~
+>>> print(packet)
+Packet({'direction': <Direction.OUTBOUND: 0>,
+ 'dst_addr': '93.184.216.34',
+ 'dst_port': 443,
+ 'icmpv4': None,
+ 'icmpv6': None,
+ 'interface': (23, 0),
+ 'ipv4': {'src_addr': '192.168.86.169',
+          'dst_addr': '93.184.216.34',
+          'packet_len': 81},
+ 'ipv6': None,
+ 'is_inbound': False,
+ 'is_loopback': False,
+ 'is_outbound': True,
+ 'payload': '\x17\x03\x03\x00$\x00\x00\x00\x00\x00\x00\x02\x05\x19q\xbd\xcfD\x8a\xe3...',
+ 'raw': <memory at 0x028924E0>,
+ 'src_addr': '192.168.86.169',
+ 'src_port': 52387,
+ 'tcp': {'src_port': 52387,
+         'dst_port': 443,
+         'syn': False,
+         'ack': True,
+         'fin': False,
+         'rst': False,
+         'psh': True,
+         'urg': False,
+         'header_len': 20,
+         'payload': '\x17\x03\x03\x00$\x00\x00\x00\x00\x00\x00\x02\x05\x19q\xbd\xcfD\x8a\xe3...'},
+ 'udp': None})
+```
+
+## Uninstalling PyDivert
+
+You can uninstall PyDivert by running
 
+```
+$ pip uninstall pydivert
+```
 
-License
-=======
+If the WinDivert driver is still running at that time, it will remove itself on the next reboot.
 
-LGPLv3
+## API Reference Documentation
 
-> This program is free software: you can redistribute it and/or modify
-> it under the terms of the GNU Lesser General Public License as published by
-> the Free Software Foundation, either version 3 of the License, or
-> (at your option) any later version.
->
-> This program is distributed in the hope that it will be useful,
-> but WITHOUT ANY WARRANTY; without even the implied warranty of
-> MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-> GNU Lesser General Public License for more details.
->
-> You should have received a copy of the GNU Lesser General Public License
-> along with this program.  If not, see <http://www.gnu.org/licenses/>.
+The API Reference Documentation for PyDivert can be found [here](FIXME).
```

Comment:
Getting the README right for pypi is a major pain in the ass, yes. :-)

FWIW, here's what it would look like in restructuredText: https://github.com/ffalcinelli/pydivert/compare/master...mhils:readme.rst?expand=1
Generally speaking I'd always go for markdown as well, but it feels like this would be bearable and we wouldn't need pandoc. No strong opinion though, so go for whatever you like best. 😊

Other than that - we're at 100% test coverage on master now. 🎉